### PR TITLE
[Reader] More events

### PR DESCRIPTION
**Pull request by NickHeiner**
Connects #2782.

[image](https://www.youtube.com/watch?v=OMOGaugKpzs)

This should be the last of the changes for the refactor. It's migrating the last of the `Analytics.events` stuff, and then adding analytics info to many more actions that were previously untracked.

### CODE REVIEW

**Comment by NickHeiner** [comment]
This is failing on CI for an unrelated reason:

image

I pinged in #appeals-engineering about it.

File: client/app/reader/actions.js [hunk 1]
```diff
@@ -108,20 +115,47 @@ export const startEditAnnotation = (annotationId) => ({
...
+  meta: {
+    analytics: {
+      category: CATEGORIES.VIEW_DOCUMENT_PAGE,
+      action: 'start-edit-annotation'
+    }
   }
 });
 
-export const openAnnotationDeleteModal = (annotationId) => ({
+export const openAnnotationDeleteModal = (annotationId, analyticsLabel) => ({
```
Comment (mdbenjam) [inline client/app/reader/actions.js]: I think you missed changing the call to this action in Comment.jsx

File: client/app/components/PdfSidebar.jsx [hunk 2]
```diff
@@ -100,6 +100,8 @@ export class PdfSidebar extends React.Component {
     }
   }
 
+  stopPlacingAnnotation = () => this.props.stopPlacingAnnotation('from-cancelling-new-annotation');
```
Comment (mdbenjam) [inline client/app/components/PdfSidebar.jsx]: We've been spelling canceling with one l.

File: client/app/reader/actions.js (hunk 1 above)
Comment (NickHeiner) [inline client/app/reader/actions.js]: ✅

**Comment by NickHeiner** [comment]
Closing so Travis will reset env vars.

**Pull request closed without merging**